ntbs-service/appsettings.Staging.json

```diff
@@ -10,10 +10,5 @@
     "AttachStackTrace": true,
     "Debug": true,
     "DiagnosticsLevel": "Error"
-  },
-  "EnvironmentDescription": {
-    "ContainsLiveData": false,
-    "IsLiveSystem": false,
-    "DisplayName": "development"
```

Comment:
Since this wasn't the same for all of our "Staging" environments I've removed it. At some point we may want to move more of our config into the `appsettings` files, but if we do so we should ensure that the environments we lump together genuinely do all want the same settings.